### PR TITLE
[Schema Registry] Initialize project azure-schemaregistry

### PR DESCRIPTION
**Pull request by YijunXieMS**
Create the bootstrap file tree for azure-schemaregistry with generated code from swagger file.
No convenience layer is added yet.
It's not in the central repo yet so Playground is used to run the CI https://dev.azure.com/azure-sdk/playground/_build?definitionId=1853&_a=summary

### CODE REVIEW

File: sdk/schemaregistry/azure-schemaregistry/swagger/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+> see https://aka.ms/autorest
+
+### Setup
+```ps
+cd C:\work
+git clone --recursive https://github.com/Azure/autorest.python.git
+cd autorest.python
+git checkout azure-core
+npm install
```
Comment (lmazuel) [inline sdk/schemaregistry/azure-schemaregistry/swagger/README.md:11]: Don't do any of those setup steps, it's deprecated

File: sdk/schemaregistry/azure-schemaregistry/swagger/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+```
+### Generation
+```ps
+cd C:\Work\SchemaRegistry\
+autorest --use=@autorest/python@5.0.0-preview.6
```
Comment (lmazuel) [inline sdk/schemaregistry/azure-schemaregistry/swagger/README.md:16]: autorest --v3 --python

File: sdk/schemaregistry/azure-schemaregistry/swagger/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+```
+### Settings
+``` yaml
+input-file: https://raw.githubusercontent.com/Azure/azure-sdk-for-java/master/sdk/schemaregistry/azure-data-schemaregistry/swagger/swagger.json
+output-folder: ../azure/schemaregistry/_generated
+namespace: azure.schemaregistry._generated
+no-namespace-folders: true
+license-header: MICROSOFT_MIT_NO_VERSION
+enable-xml: false
```
Comment (lmazuel) [inline sdk/schemaregistry/azure-schemaregistry/swagger/README.md:25]: That's not a thing in Python

File: sdk/schemaregistry/azure-schemaregistry/swagger/README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+vanilla: true
```
Comment (lmazuel) [inline sdk/schemaregistry/azure-schemaregistry/swagger/README.md:26]: That's not a thing in Pthon

File: sdk/schemaregistry/azure-schemaregistry/swagger/README.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+npm install
+```
+### Generation
+```ps
+cd C:\Work\SchemaRegistry\
+autorest --use=@autorest/python@5.0.0-preview.6
+```
+### Settings
+``` yaml
+input-file: https://raw.githubusercontent.com/Azure/azure-sdk-for-java/master/sdk/schemaregistry/azure-data-schemaregistry/swagger/swagger.json
```
Comment (lmazuel) [inline sdk/schemaregistry/azure-schemaregistry/swagger/README.md:20]: Let's make sure we use a Swagger from the spec repo eventually

File: sdk/schemaregistry/azure-schemaregistry/swagger/README.md (hunk 1 above)
Comment (YijunXieMS) [reply sdk/schemaregistry/azure-schemaregistry/swagger/README.md]: Removed the setup section

File: sdk/schemaregistry/azure-schemaregistry/swagger/README.md (hunk 2 above)
Comment (YijunXieMS) [reply sdk/schemaregistry/azure-schemaregistry/swagger/README.md]: Updated

File: sdk/schemaregistry/azure-schemaregistry/swagger/README.md (hunk 5 above)
Comment (YijunXieMS) [reply sdk/schemaregistry/azure-schemaregistry/swagger/README.md]: Updated to use https://raw.githubusercontent.com/Azure/azure-rest-api-specs/arerlend.sr.init/specification/schemaregistry/data-plane/Microsoft.EventHub/preview/2018-01-01-preview/schemaregistry.json.
Will update again once it's merged to master.

File: sdk/schemaregistry/azure-schemaregistry/swagger/README.md (hunk 4 above)
Comment (YijunXieMS) [reply sdk/schemaregistry/azure-schemaregistry/swagger/README.md]: Removed.

File: sdk/schemaregistry/azure-schemaregistry/swagger/README.md (hunk 3 above)
Comment (YijunXieMS) [reply sdk/schemaregistry/azure-schemaregistry/swagger/README.md]: Removed.

**Pull request merged**